source/newcommitter.md

```diff
@@ -20,11 +20,46 @@ The [Contributor Ladder](https://community.apache.org/contributor-ladder.html) h
 
 {{% toc %}}
 
+<a name="NewCommitter-Summary"></a>
+### TL;DR - Inviting a new Committer
+
+1. Discuss the proposed committer (and optionally PMC member). (Use
+   [this template](templates/committer-discuss.txt).)
+If the discussion seems to be going positively,
+   call a vote. (Use [this template](templates/committer-vote.txt) for
+   committer, [this template](templates/committer-pmc-vote.txt) for
+   committer and PMC member.)
+1. Close the vote. (Use [this template](templates/committer-results.txt) for a committer vote, or [this template](templates/committer-pmc-results.txt) for a committer + PMC member vote.)
+1. If the result is positive, invite the new committer. (Use [this
+   template](templates/committer-invite.txt).)
+
+If they accept, then:
+
+1. If they already have an Apache id, grant appropriate commit privileges.
+   Use the Whimsy tool to update the roster via [Comitee Roster](https://whimsy.apache.org/roster/committee/) or
+   [PPMC Roster](https://whimsy.apache.org/roster/ppmc/)
+1. If they have already filed an ICLA, request creation of the committer account.
+   If they need to change anything in a previously filed ICLA, wait until the new ICLA is filed,
+   then request the account.
+   1. Wait until root says it is done
+   1. PMC Chair updates LDAP group membership which enables svn, gitbox and other access.
+      If the committer uses GitHub, they are responsible for linking it to their ASF account.
+   1. Add committer to the appropriate groups in Jira and CWiki
```

Comment:
I believe (but I am not 100% sure) - we do not have to add committers or PMC members to JIRA and Cwiki groups - this is happening automatically via LDAP addition. At least I don't remember - over last few years - that I had to do that - we can give permissions to "airflow" and "airflow-pmc" via https://cwiki.apache.org/confluence/spaces/spacepermissions.action?key=AIRFLOW - but I don't recall a way to edit those groups and add new people there.  I looked for it today, but I could not find it.

But maybe I am simply not aware of that- we don't use JIRA and  for CWIKI we usualy add permissions to people individually when they ask for an account. And I am not even sure if the accounts in CWiki and Commiter IDs are connected in any way. So I think that part is a little blurry.

Reply:
Yeah, that bit was part of the original doc, and I think has been around for a long time, like the stuff about svn. Definitely need to bring it into the current decade.

Reply:
Fixed.